industry_benchmarks/utils/data_gathering.py

```diff
@@ -0,0 +1,422 @@
+import click
+import pathlib
+import json
+import rdkit
+from rdkit import Chem
+from rdkit.Chem import AllChem
+import gufe
+import openfe
+from openfe import LigandNetwork
+from kartograf.atom_mapping_scorer import (
+    MappingRMSDScorer, MappingShapeOverlapScorer, MappingVolumeRatioScorer,
+)
+
+
+def parse_ligand_network(
+    input_ligand_network_path: str,
+) -> LigandNetwork:
+    """
+    Read a LigandNetwork
+    """
+    with open(input_ligand_network_path) as f:
+        graphml = f.read()
+
+    network = gufe.LigandNetwork.from_graphml(graphml)
+    return network
+
+
+def get_blinded_transformation_network(
```

Comment:
Something to be discussed, we told the Pharma partners to use ligand names that they can disclose, so they should have blinded them themselves. If we wanted to blind them, we'd have to do that in many different places, since we use the ligand names in most files.